src/Indigo/Contracts/HomebaseLite/Types/FA2.hs

```diff
@@ -0,0 +1,22 @@
+-- SPDX-FileCopyrightText: 2022 Tezos Commons
+-- SPDX-License-Identifier: LicenseRef-MIT-TC
+
+module Indigo.Contracts.HomebaseLite.Types.FA2
+  ( BalanceRequestItem(..)
+  , BalanceResponseItem(..)
+  , FA2Parameter(..)
+  , TokenId(..)
+  ) where
+
+import Indigo
+
+import Lorentz.Contracts.Spec.FA2Interface
+
+data FA2Parameter
```

Comment:
Was it necessary to add this type?

Couldn't we instead used the pre-defined `Parameter` in `Lorentz.Contracts.Spec.FA2Interface` (and the instances in `Lorentz.Contracts.Spec.FA2Interface.ParameterInstances`)?

Reply:
Unnecessary, indeed. I remember I had _some_ issue with the definition from `FA2Interface`, but can't exactly recall what that was, and I didn't encounter it now.